SOFTWARE/AfterlifeLightKit/ConfigManager.cpp

```diff
@@ -0,0 +1,115 @@
+#include <FS.h>
+#include <LittleFS.h>
+#include <EEPROM.h>
+#include <ArduinoJson.h> // https://github.com/bblanchon/ArduinoJson
+
+#include "ConfigManager.h"
+
+void ConfigManager::init()
+{
+    if (!LittleFS.begin())
+    {
+        /*
+         * TODO: If this error occurs, we should still
+         * continue executing, but the Wifi Configuration feature
+         * will be unavailable. Return a hard-coded HTML error page?
+         */
+        Serial.println("An Error has occurred while mounting LittleFS");
+        return;
+    }
+
+    File configFile = LittleFS.open("/config.json", "r");
+
+    if (!configFile)
+    {
+        Serial.println("Opening config file failed.");
+        return;
+    }
+
+    // Allocate a temporary JsonDocument
+    // Don't forget to change the capacity to match your requirements.
+    // Use arduinojson.org/v6/assistant to compute the capacity.
+    StaticJsonDocument<512> doc;
+    if (configFile.available())
+    {
+        DeserializationError error = deserializeJson(doc, configFile);
+        if (error) {
+            Serial.println(F("Failed to read file, using default configuration"));
+        }
+    }
+
+    configFile.close();
+
+    /**
+     * TODO: How do we abstract this out?
+     */
+    _hardwareConfiguration.Powercell.length = doc["hardware"]["powercell"]["length"] | 15;
+    _hardwareConfiguration.Cyclotron.length = doc["hardware"]["cyclotron"]["length"] | 60;
+
+    //_configuration.classicSettings.powercellSpeed = doc["modes"]["classic"]["powercell"]["speed"] | 1000;
+
+
+    _configuration.afterlifeSettings.powercell.speed = doc["modes"]["afterlife"]["powercell"]["speed"] | 50;
```

Comment:
We might be able to rewrite this to set all the `afterlifeSettings` at once from the JSON, especially if every "mode" has the exact same settings available (speed, color, direction, etc).